mono/utils/mono-threads-debug.h

```diff
@@ -6,11 +6,19 @@
 #define __MONO_UTILS_MONO_THREADS_DEBUG_H__
 
 /* Logging - enable them below if you need specific logging for the category you need */
+#ifdef HOST_WIN32
+#define MOSTLY_ASYNC_SAFE_PRINTF(...) do { \
+	char __buff[1024];	__buff [0] = '\0'; \
+	g_snprintf (__buff, sizeof (__buff), __VA_ARGS__);	\
+	_write (1, __buff, (guint32)strlen (__buff));	\
```

Comment:
Add `g_write`?

Reply:
Do you mean that we should add a `g_write` function to do this? I don't see one that exists currently.

Reply:
Yes, the same way we have `g_open` with
```
#ifdef G_OS_WIN32
#define g_write _write
#else
#define g_write write
#endif
```

Reply:
Ok, I'll do that.

Reply:
This is a debug macro, it's fine either way.

Reply:
I've fixed this at a0c1aa4b06545cc9c738472ad7c6175def3b148a.